Game/assets/prefabs/generated/11820935048415628422.prefab.meta

```diff
@@ -1,5 +1,5 @@
-asset_hash: 14590907171513910164
+asset_hash: 5303512136444773407
```

Comment:
Dont include these random hash changes for prefabs and for scenes in the commit if possible. They are not needed and it keeps everything cleaner & Easier to merge (I know its annoying)